README.md

```diff
@@ -34,11 +34,11 @@ This project project was built following the specifications , and user storiess
 ## Presentation/Demo:
 
 - Video-presentation of the project
```

Comment:
Kindly change this to a live version of the project. There is no video presentation, the link when tested is the live version of the project, although the link does not show on the README

Reply:
Issue solved. Thank you for your feedback